Makefile

```diff
@@ -207,6 +207,10 @@ proto-mocks: protoc
 		mockgen -copyright_file ../LICENSE -package $(call service_name,$(PROTO_GRPC_SERVICE))mock -source $(PROTO_GRPC_SERVICE) -destination $(call mock_file_name,$(PROTO_GRPC_SERVICE)) \
 	$(NEWLINE))
 
+proto-clients:
```

Comment:
Nitpicky, but I don't think this name makes sense. grpc already generates "proto clients", as in client stubs for services defined in proto[buf] files. The code in `client` adds metrics, retries (should be moved to interceptors), redirection, and most importantly routing, which are all application-level stuff, not "proto" stuff.

Maybe `service-clients`?

Reply:
I was going back and forth on the name, I like `service-clients`.